planning/behavior_path_planner/CMakeLists.txt

```diff
@@ -7,8 +7,6 @@ autoware_package()
 find_package(OpenCV REQUIRED)
 find_package(magic_enum CONFIG REQUIRED)
 
-set(COMPILE_WITH_OLD_ARCHITECTURE TRUE)
```

Comment:
I think it it better to remain this line and set the flag `FALSE`.

Reply:
Thanks.
Fixed at [3494fc9](https://github.com/tier4/autoware.universe/pull/387/commits/3494fc91f343195ba1f528b848f944b116d7fc76)